### ISSUE TITLE
Carriage return and line endings in index.js is broken on latest update

### ISSUE THREAD

**Issue by vivekgalatage** [charlesguse/run-script-os#28]
The recent change (which has been committed) has a different line endings in `index.js` and due to which on ubuntu 18.04, we are getting the below error:

```
> run-script-os

/usr/bin/env: ‘node\r’: No such file or directory
npm ERR! code ELIFECYCLE
npm ERR! syscall spawn
npm ERR! file sh
npm ERR! errno ENOENT
npm ERR! vision@0.1.0 cmdtools_download: `run-script-os`
npm ERR! spawn ENOENT
npm ERR! 
npm ERR! Failed at the vision@0.1.0 cmdtools_download script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-03-03T15_19_54_083Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! vision@0.1.0 setup: `npm run cmdtools_download && npm run sdk_manager_download`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the vision@0.1.0 setup script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/user/.npm/_logs/2020-03-03T15_19_54_092Z-debug.log
```

**Comment by vivekgalatage** [comment]
Running `dos2unix` on the `node_modules/run-script-os/index.js` fixes the problem... The line endings from the merge request needs to be fixed

**Comment by charlesguse** [comment]
This should be fixed in v1.1.1.
